ConverterViewController.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  ConverterViewController.swift
+//  anarmyofones
+//
+//  Created by Fabian Mariño on 4/4/16.
+//  Copyright © 2016 Fabian Mariño. All rights reserved.
+//
+
+import UIKit
+
+// MARK: Local Constants
+
+// Define server side script URL
+let EXCHANGESURL = "https://api.fixer.io/latest?base=USD&symbols=GBP,EUR,JPY,BRL"
+let REQUESTMETHOD = "GET"
+let CURRENCIES = [ "UK": "GBP",
+                   "EUR": "EUR",
+                   "JAP": "JPY",
+                   "BRA": "BRL"
+                 ]
+let FIELDMAXLENGTH = 8
+let DEFAULTAMOUNT = 1.0
+
+class ConverterViewController: UIViewController, UITextFieldDelegate {
+    
+    var _dollarAmount: Double? = DEFAULTAMOUNT
+    var _exchangesDictionary:[String:Double]?
+    let _currencyFormatter : NSNumberFormatter = NSNumberFormatter()
+    var _requestedWasOk = false
+    
+    @IBOutlet weak var lblGBP: UILabel!
+    @IBOutlet weak var lblEUR: UILabel!
+    @IBOutlet weak var lblBRL: UILabel!
+    @IBOutlet weak var lblJPY: UILabel!
+    
+    @IBOutlet weak var dollarNumberTextField: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //sets format number style to currency
+        _currencyFormatter.numberStyle = NSNumberFormatterStyle.CurrencyStyle
+        
+        //set delegate for dollarnumberUITextField
+        self.dollarNumberTextField.delegate = self
+        
+        //get current dollar price from fixer.io
+        getCurrencyValues(self.updateValues)
+        
+        // Gesture that recognices tap to hide keyboard
+        let tapGesture = UITapGestureRecognizer(target: self, action:Selector("endEditing:"))
+        tapGesture.cancelsTouchesInView = false
+        self.view.addGestureRecognizer(tapGesture);
+    }
+    
+    // fetches exchange values for dollar using a get request and receiving a callback
+    func getCurrencyValues(completion: (() -> Void)!) {
```

Comment:
A nice thing you can do here, is creating a `type alias` for the closure.
`typealias CompletionClosure = () -> Void`
And then this would look like: `func getCurrencyValues(completion: CompletionClosure!)`
BUT you should not use force unwrapping so better go with `?`